cypress/e2e/people-to-notify.cy.js

```diff
@@ -1,3 +1,5 @@
+import {AddressFormAssertions} from "../support/e2e";
```

Comment:
I've left the format of this test relying on fixture objects intact for now until we tackle the fixtures ticket as we're still undecided on how this will work in practice.